### PR TITLE
Helvetica regular

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Who is this for?
Experience team initially. Maybe everyone who longs for legible text on the site, ultimately.

## What is it doing for them?
Giving them the option to swap out Helvetica light for regular via the toggles dash.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
If we've concluded it's more legible, shouldn't we just change it for everyone?

**Comment by davidpmccormick** [comment]
We haven’t concluded anything — we’re putting it behind a toggle to so we can see how it looks

File: common/views/pages/_app.js [hunk 1]
```diff
@@ -399,6 +399,28 @@ export default class WecoApp extends App {
...
+                      <TogglesContext.Consumer>
+                        {({ helveticaRegular }) =>
+                          helveticaRegular && (
+                            <style
+                              type="text/css"
+                              dangerouslySetInnerHTML={{
+                                __html: `
+                                @font-face {
+                                  font-family: 'Helvetica Neue Light Web';
+                                  src: local('Helvetica Neue Light'),
```
Comment (gestchild) [inline common/views/pages/_app.js]: Is this local source still meant to be Helvetica Neue Light?

Comment (davidpmccormick) [reply common/views/pages/_app.js]: Nope. Good spot, thanks. Will update.

**Pull request merged**